src/main/java/com/minecolonies/inventory/InventoryCitizen.java

```diff
@@ -393,7 +402,13 @@ public boolean isSlotEmpty(int index)
         return getStackInSlot(index) == null;
     }
 
-    /**
+    public void createMaterialStore(@NotNull MaterialSystem system)
+    {
+        if (materialStore == null)
+        {
+            materialStore = new MaterialStore(MaterialStore.Type.INVENTORY, system);
+        }
+    }    /**
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ATrailingCommentCheck)